src/pages/patch/patchTabs/CodeChanges.tsx

```diff
@@ -0,0 +1,130 @@
+import React from "react";
+import { useQuery } from "@apollo/react-hooks";
+import { useParams } from "react-router-dom";
+import { Skeleton, Table } from "antd";
+import { H2 } from "components/Typography";
+import { uiColors } from "@leafygreen-ui/palette";
+import {
+  GET_CODE_CHANGES,
+  GetCodeChangesQuery,
+  FileDiff
+} from "gql/queries/get-code-changes";
+import Button from "@leafygreen-ui/button";
+import styled from "@emotion/styled";
+
+export const CodeChanges = () => {
+  const { id } = useParams<{ id: string }>();
+  const { data, loading, error } = useQuery<GetCodeChangesQuery>(
+    GET_CODE_CHANGES,
+    {
+      variables: { id }
+    }
+  );
+  if (loading) {
+    return <Skeleton active={true} title={true} paragraph={{ rows: 8 }} />;
+  }
+  if (error) {
+    return <div id="patch-error">{error.message}</div>;
+  }
+  if (!data.patch.moduleCodeChanges.length) {
+    return <Title className="cy-no-code-changes">No code changes</Title>;
```

Comment:
one of the best practices listed in the cypress docs is using `data-cy` attributes instead of IDs and classes. thought we could start doing this going forward.